src/utils/constants.js

```diff
@@ -3,12 +3,19 @@ import EnvLib from '@adobe/aio-lib-env'
 const { getCliEnv } = EnvLib
 const clientEnv = getCliEnv()
 
+const COMMON = {
+  // need to update this to the correct official URL
+  REMOTE_ACCS_MESH_CONFIG: 'https://gist.githubusercontent.com/revanth0212/1c83bd4ad0be161c209cef07722df04c/raw'
```

Comment:
Need to update this before merging

Reply:
FYI you can get a raw, non-hashed url if you create a git repo:

For example: https://raw.githubusercontent.com/adobe-summit-L445/imaging/main/checkout.sh